src/Adapter/Order/QueryHandler/GetOrderForViewingHandler.php

```diff
@@ -448,9 +455,9 @@ private function getOrderProducts(Order $order): OrderProductsForViewing
                 $totalPriceFormatted,
                 $product['current_stock'],
                 $imagePath,
-                Tools::ps_round($product['unit_price_tax_excl'], 2),
-                Tools::ps_round($product['unit_price_tax_incl'], 2),
-                $product['amount_refund'],
+                $product['unit_price_tax_excl'],
```

Comment:
However here since the value is returned as raw float (not displayed price) you can use Tools::ps_round with appropriate computing precision Try to match what has been done in this PR https://github.com/PrestaShop/PrestaShop/pull/16150 as it will avoid conflicts

https://github.com/PrestaShop/PrestaShop/blob/03598a6060909e84f035393a16822c1b559b43b5/src/Adapter/Order/QueryHandler/GetOrderForViewingHandler.php#L448